modules/jdbc/src/main/java/org/testcontainers/containers/JdbcDatabaseContainer.java

```diff
@@ -107,8 +107,8 @@ public SELF withConnectTimeoutSeconds(int connectTimeoutSeconds) {
         return self();
     }
 
-    public SELF withInitScript(String initScriptPath) {
-        this.initScriptPath = initScriptPath;
+    public SELF withInitScript(String... initScriptPath) {
```

Comment:
this breaks binary compatibility. Please make sure that the old method (`withInitScript(String)`) works as before

Reply:
Hello @bsideup,
Thanks for your feedback.
I've merged some changes from @jbek7 into this pull request that introduces a separate method `withMultiInitScript` to accept multiple sql files.

Reply:
@sfodje @bsideup
thanks.